boards/microchip/mec_assy6941/board.yml

```diff
@@ -0,0 +1,9 @@
+board:
+  name: mec_assy6941
+  full_name: MEC174x/5x EVB
```

Comment:
To follow the naming of other boards I think it might be better to do as follows?

```suggestion
  full_name: MEC17xxEVB ASSY6941
```

Reply:
Done. Pushed update.